libs/libcommon/include/common/avx2_mem_utils.h

```diff
@@ -136,56 +141,78 @@ FLATTEN_INLINE_PURE static inline bool check_block32x4_eq(const char * a, const
         auto all_ones = _mm256_set1_epi8(0xFF);
         Block32 data = all_ones;
         for (size_t i = 0; i < AVX2_UNROLL_NUM; ++i)
-            data = _mm256_and_si256(data, _mm256_cmpeq_epi8(load_block32(a + i * BLOCK32_SIZE), load_block32(b + i * BLOCK32_SIZE)));
+            data = _mm256_and_si256(data, _mm256_cmpeq_epi8(load_block32(b + i * BLOCK32_SIZE), load_block32(a + i * BLOCK32_SIZE)));
```

Comment:
Why change this?

Reply:
A small typo but won't affect correction.
Fixed.